### PR TITLE
Poc/validation for name places accents

### PR DESCRIPTION
**Pull request by rmothilal**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -72,7 +74,10 @@ const initServer = async function (db, config) {
     port: config.listenPort,
     routes: {
       validate: {
-        failAction: failActionHandler
+        options: ErrorHandler.validateRoutes(),
+        failAction: async (request, h, err) => {
```
Comment (lewisdaly) [inline src/server.js]: I think I just had `failActionHandler` here for the improved test coverage it offered by abstracting it out. No biggie though.

File: src/model/quotes.js [hunk 2]
```diff
@@ -1071,7 +1071,7 @@ class QuotesModel {
   generateRequestHeaders (headers, noAccept) {
     const ret = {
       'Content-Type': 'application/vnd.interoperability.quotes+json;version=1.0',
-      Date: headers.date,
+      Date: (headers.date && (typeof headers.date !== 'string')) ? headers.date.toUTCString() : headers.date,
```
Comment (lewisdaly) [inline src/model/quotes.js]: I don't think I saw a test for this, but I think you said you'd worry about tests and coverage later?

File: src/server.js (hunk 1 above)
Comment (rmothilal) [reply src/server.js]: ok that's cool i can move it out to that util file

File: src/model/quotes.js (hunk 2 above)
Comment (rmothilal) [reply src/model/quotes.js]: yup
there is a negative test, but will do the positive with the missing route tests

Comment (rmothilal) [reply src/model/quotes.js:1074]: Have added the extra tests for this

File: audit-resolve.json [hunk 3]
```diff
@@ -1,39 +1,39 @@
 {
   "decisions": {
-    "1482|@hapi/hapi": {
-      "decision": "ignore",
-      "madeAt": 1582635042880,
-      "expiresAt": 1583239828928
-    },
     "1179|@mojaloop/central-services-shared>@mojaloop/event-sdk>grpc>node-pre-gyp>mkdirp>minimist": {
```
Comment (lewisdaly) [inline audit-resolve.json:3]: This is a real pain. FYI I did some research on this issue, and it looks like `node-pre-gyp` simply hasn't been updated to the latest version of `mkdirp`, which contains this issue.

Comment (rmothilal) [reply audit-resolve.json:3]: yeah its a pain, the only thing we can do till its fixed is to ignore the issue, which isn't the best approach

**Comment by rmothilal** [comment]
This is not the approved method for validation. closing

**Pull request closed without merging**